spec/message_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'message'
+require 'dotenv'
+require 'twilio-ruby'
+Dotenv.load
+
+
+describe Message do
+  let(:client) { double :client, messages: messages }
+  let(:messages) { double :messages }
```

Comment:
You have initialized and assigned messages after it was used in client on line 8